scalafmt-tests/src/test/resources/trailing-commas/trailingCommasAlways.stat

```diff
@@ -358,10 +358,12 @@ import foo.{
   a, b,
 }
 class foo(
-    a: String, b: String,
+    a: String,
```

Comment:
this and all other "trailing commas" changes are simply a fix: all have `allowFolding = false` hence must preserve dangling close paren which is enough to output config-style (since dangling parens are off by default in the tests)